src/Sylius/Bundle/ShopBundle/Resources/views/Product/Show/_price.html.twig

```diff
@@ -2,6 +2,20 @@
 
 {% set variant = product|sylius_resolve_variant %}
 {% set hasDiscount = variant|sylius_has_discount({'channel': sylius.channel}) %}
+{% set channelPricing = variant.channelPricings.first %}
```

Comment:
you shoudn't get here first `channelPricing` but channel pricing for active channel 
```suggestion
{% set channelPricing = variant.getChannelPricingForChannel(sylius.channel %}
```